packages/eslint-plugin/src/rules/strict-boolean-expressions.ts

```diff
@@ -715,11 +715,6 @@ export default util.createRule<Options, MessageId>({
             context.report({
               node,
               messageId: 'conditionErrorNullableObject',
-              fix: util.getWrappingFixer({
-                sourceCode,
-                node,
-                wrap: code => `${code} != null`,
-              }),
```

Comment:
Could you move this to a [_suggestion_ (`suggest`)](https://eslint.org/docs/latest/extend/custom-rules#providing-suggestions) instead? It's useful to offer this to users, just we can't auto-apply it under `--fix`.